controllers/chaosimpl/physicalmachinechaos/impl.go

```diff
@@ -43,20 +44,35 @@ type Impl struct {
 func (impl *Impl) Apply(ctx context.Context, index int, records []*v1alpha1.Record, obj v1alpha1.InnerObject) (v1alpha1.Phase, error) {
 	impl.Log.Info("apply physical machine chaos")
 
-	physicalMachinechaos := obj.(*v1alpha1.PhysicalMachineChaos)
-	address := records[index].Id
+	physicalMachineChaos := obj.(*v1alpha1.PhysicalMachineChaos)
+	var address string
+	if len(physicalMachineChaos.Spec.Address) > 0 {
+		address = records[index].Id
+	} else {
+		var physicalMachine v1alpha1.PhysicalMachine
+		namespacedName, err := controller.ParseNamespacedName(records[index].Id)
+		if err != nil {
+			return v1alpha1.NotInjected, err
+		}
+		err = impl.Get(ctx, namespacedName, &physicalMachine)
+		if err != nil {
+			// TODO: handle this error
+			return v1alpha1.NotInjected, err
+		}
+		address = physicalMachine.Spec.Address
+	}
```

Comment:
Please append some comments about the output of physical machine selector like "when using `address` directly, values in records[] are IP, when using selector, values in records are NamespacedName"

Reply:
done